ci/shell.nix

```diff
@@ -88,7 +88,7 @@ stdenvNoCC.mkDerivation ({
     openssl
     pkgconfig
     poetry
-    protobuf3_6
+    protobuf3_19
```

Comment:
is protobuf always versioned? couldn't we just use `protobuf` or something?

Reply:
We can use `protobuf` which is the latest packaged version (3.19). Don't know if there was a reason to use explicit version in the first place.